recipes/gdal/all/conanfile.py

```diff
@@ -404,6 +404,13 @@ def _patch_sources(self):
                                       "-lz ",
                                       "-l{} ".format(zlib_name))
 
+        # Redhat 7 needs an existing config.rpath file
+        major_version = str(tools.os_info.os_version.major(False))
+
+        if tools.os_info.linux_distro in ["rhel", "centos"] and major_version == "7":
+            with open(os.path.join(self._source_subfolder,"config.rpath"),"w"):
+                pass
+
```

Comment:
As I said in https://github.com/conan-io/conan-center-index/issues/5707, it's not specific to an OS. This issue is a consequence of https://github.com/conan-io/conan-center-index/pull/5539 I think.